src/db.py

```diff
@@ -25,8 +39,32 @@
 
 def get_campus_eateries(data_json, refresh=False):
     if refresh:
-        Base.metadata.drop_all(bind=Engine)
-        Base.metadata.create_all(bind=Engine)
+        Base.metadata.drop_all(
```

Comment:
Could I get a summary on how this fixes the item duplication issue we just saw?

Also, is this fix a temporary one?

Reply:
Before the change, db tables that were unrelated to the corresponding update operations were being modified so this fix makes sure that only the tables directly related to the operation is modified.